package.json

```diff
@@ -11,7 +11,7 @@
     "test": "npm run test:server && npm run test:browser",
     "test:server": "mocha \"test/**/*.coffee\"",
     "test:browser": "mochify \"test/**/*.coffee\" --transform=coffeeify --extension=.coffee",
-    "prepublish": "npm run build"
+    "prepublishOnly": "npm run build"
```

Comment:
Using `prepublishOnly` runs this only when publishing the package. Using `prepublish` runs this after each `npm install`.